### DISCUSSION TITLE
All Payload Choices Output a .bin file?

### DISCUSSION THREAD

**Discussion by xkillbit** [GeorgePatsias/ScareCrow-CobaltStrike#9] [Q&A]
Hello, I set up my CS server following the instructions in the Readme.md. When I generate any payload from the ScareCrow Payload Generator inside Cobalt Strike. I always get a scbeacon.bin file. Is this expected? What am I doing wrong? Is there some additional step I should be taking?

**Comment by sidonpc** [comment]
Same issue here...
